client/src/components/MoreHomes.jsx

```diff
@@ -20,8 +30,9 @@ class MoreHomes extends React.Component {
     let key = Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
This should probably be read from the url rather than randomly created.

Reply:
Fixed!